### ISSUE TITLE
Voice line system

### ISSUE THREAD

**Issue by SCIENTIST2004** [BEEmod/BEE2.4#1330]
can you make it so that users themselves choose the playback order of audio files?
This will allow users to create their own monologues and even use more than one vote.
![Гифка с Gifius ru](https://user-images.githubusercontent.com/35931028/85228214-dd9d1580-b3ea-11ea-858d-057f1c744aa5.gif)
You can also listen to it in advance. Like music

**Comment by CatzRule81** [comment]
Of course, the lines would have to already exist in Portal 2 to be used. (Unless you have vastly superior modding experience)

With this feature there might also be the possibility to have multiple characters speaking in the same dialog chain. Though this might be excessively hard to replicate. It should be doable enough though

**Comment by unfunny1** [comment]
yes please i need it enderek sound trigger is such a pain in the ass to use
